client/src/components/Welcome.tsx

```diff
@@ -87,8 +87,8 @@ const Welcome = (): JSX.Element => {
             <div className={`rounded-br-2xl ${commanStyles}`}>Blockchain</div>
           </div>
         </div>
-        <div className="flex flex-col flex-1 items-end justify-start w-full md:mt-0 mt-10  ">
-          <div className="p-3 justify-end items-start flex-col rounded-xl h-48 sm:w-80 w-full  max-w-sm my-5 eth-card white-glassmorphism hover:shadow-2xl shadow-cyan-600">
+        <div className="flex flex-col flex-1 items-center md:items-end justify-start w-full md:mt-0 mt-10  ">
```

Comment:
Please change `items-center` for all screens widths

Reply:
okay